packages/core/src/components/fields/SchemaField.tsx

```diff
@@ -249,6 +249,12 @@ function SchemaFieldRender<T, F>(props: FieldProps<T, F>) {
   if (!hideError && errors && errors.length > 0) {
     classNames.push("field-error has-error has-danger");
   }
+  if (uiSchema?.classNames) {
+    console.warn(
```

Comment:
I think `console.warn` might be called hundreds of times , due to each re-render? This has caused issues with deprecation warnings in the past as they slow down things immensely. We might also want to only call this warning if `process.env.NODE_ENV === 'development'`

Reply:
I'll add the `process.env.NODE_ENV`... as for slowing things down, well there is a clear fix to making them go away ;)